app/views/documents/_document.html.erb

```diff
@@ -1,4 +1,9 @@
-<div class="uploaded_document">
-  <%= f.mb_input_field :document, nil,
-    options: { value: File.basename(document), disabled: true } %>
+<div class="doc-preview">
+  <div class="doc-preview__info">
+    <h4><%= File.basename(url || '{{filename}}') %></h4>
```

Comment:
Yeah, it's weird. Best way to describe its purpose is that it's a placeholder for handlebars.js to use to interpolate.

So, if we have a url, get it's `basename`, otherwise output `'{{filename}}'` so handlebars can replace it.

What do you think about putting it up at the top in some ruby like:

```ruby
<%
doc_title = if url
              File.basename(url)
            else
              "{{filename}}"
            end
%>

... ✂ snip ✂ ...

<h4><%= doc_title %></h4>
```